macOS/DuckDuckGo/Fire/View/FirePopoverViewController.swift

```diff
@@ -216,7 +216,7 @@ final class FirePopoverViewController: NSViewController {
             assertionFailure("No TabCollectionViewModel or MainWindowController")
             return
         }
-        windowController.window?.performClose(self)
```

Comment:
performClose may is routed through the window->closeButton->click and may be ignored